src/components/routine/RoutineForm.tsx

```
@@ -72,83 +45,134 @@ const RoutineForm = ({ nickname, mateNickname }: RoutineFormProps) => {
+            min={1}
+            placeholder="루틴 횟수를 입력하세요."
+            onChange={onChange}
+          />
+        )}
+      />
+      <FormItem
+        name="startDate"
+        className="flex flex-col gap-2 mt-5"
+        label="루틴 시작 날짜"
+        rule={{ required: true }}
+        render={({ value, name, onChange, form, setValue }) => (
```

ESLint

[error] 'form' is already declared in the upper scope on line 30 column 9.

(@typescript-eslint/no-shadow)